lib/monitoring/check_result.rb

```diff
@@ -0,0 +1,30 @@
+module Monitoring
+  class CheckResult
```

Comment:
This now captures the contract between checker and notifier.  Before it was basically whatever and so it resulted in tight coupling between some checkers and notifiers.